src/saving/SaveList.cs

```diff
@@ -75,6 +75,9 @@ public class SaveList : ScrollContainer
     private PackedScene listItemScene = null!;
 #pragma warning restore CA2213
 
+    private int allItemsCount = 0;
+    private int loadedItems = 0;
```

Comment:
These items do not need default values, another reason why checks are complaining